cmd/monaco/delete/delete.go

```diff
@@ -83,14 +89,35 @@ func deleteConfigs(environments []manifest.EnvironmentDefinition, apis api.APIs,
 
 func deleteConfigForEnvironment(env manifest.EnvironmentDefinition, apis api.APIs, entriesToDelete map[string][]delete.DeletePointer) []error {
 	dynatraceClient, err := dynatrace.CreateDTClient(env.URL.Value, env.Auth, false)
-
 	if err != nil {
 		return []error{
 			fmt.Errorf("It was not possible to create a client for env `%s` due to the following error: %w", env.Name, err),
 		}
 	}
 
+	var autClient *automation.Client
```

Comment:
Why not move clientset.go a layer up in to the client package and call `CreateClientSet()` ?`

Reply:
See PR description - I wanted to keep "delete automations" separate from big refactoring. 
The clientSets in several places and the fact that each cmd sets up it's own automation client I'd change into a unified setup in the 'DynatraceClient' as a followup